### ISSUE TITLE
Czkawka crashes as soon I start scanning for similar images.

### ISSUE THREAD

**Issue by Erothas** [qarmin/czkawka#618]
**operating system**
Linux
**GUI or CLI**
GUI
![selection-Fri_28_Jan_11 58 49_CET_2022](https://user-images.githubusercontent.com/55455670/151536408-df4b4dfd-22b8-4e87-8a63-a1ee01d2129a.png)
This is what I get as soon as I started scanning similar images. Do note that lower hash sizes works fine.

**Comment by qarmin** [comment]
I was think that new feature to open images in closure will save from such crashes, but looks that it doesn't work for all types of bugs.

The problem is caused by opening image(which try to allocate several thousands of GB).
Problem is already fixed in 0.24 version of image-rs library, but it is not released yet

I suggest to check with tool which files cause problems/crashes and move them out of scanned directory:
```
cargo install --git https://github.com/qarmin/ImageOpening --branch 0.23
image_opening /home
```

**Comment by Erothas** [comment]
Oh no.. Not another thing I have to wait image-rs to release.. Damn it!
I guess I have to wait a millennium until they release that fix alongside the gif fix that has been done another millennium ago.
